_vendors.yml

```diff
@@ -172,3 +172,13 @@ creative_commons:
   version: 2020.11.3
   dir: assets/license_badges
   alias: creativecommons-vocabulary
+wavedrom:
+  name: 'wavedrom'
```

Comment:
You can remove single quotes in this line

Reply:
got it. removed.